### PR TITLE
Updated README to include fallback.

### PR DESCRIPTION
**Pull request by Himujjal**
Hi. Great repo. But I can't seem to work with the npx command. So, I updated the README.md to include a fallback for temporary purposes in case someone couldn't figure out what to do. This is of course temporary.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,11 @@ npm i @asgerf/dts-tree-sitter
 
 npx @asgerf/dts-tree-sitter INPUT > OUTPUT.d.ts
 ```
+If the above command fail:
```
Comment (asgerf) [inline README.md]:
```suggestion
Alternative if you prefer to run without `npx`:
```

File: README.md [hunk 2]
```diff
@@ -19,11 +24,18 @@ where `INPUT` is used to locate a `node-types.json` file in one of the following
 ## Example
 
 The `tree-sitter-javascript` grammar can be compiled like this:
+
 ```sh
 npm i tree-sitter-javascript
 npx @asgerf/dts-tree-sitter tree-sitter-javascript > generated.d.ts
 ```
 
+Fallback to the above command:
```
Comment (asgerf) [inline README.md]:
```suggestion
Alternative if you prefer to run without `npx`:
```

**Review by asgerf — COMMENTED** [review]
Suggesting a different phrase to make it clear that the fallback relates to `npx`

**Comment by Himujjal** [comment]
cool. that can be merged.

**Comment by razzeee** [comment]
Thx, had to fallback to this for inclusing in the package.json scripts of my project

**Pull request merged**